content/en/Platform Deep Dive/Collaboration/manage-notifications.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Manage Notifications"
+linkTitle: "Manage Notifications"
+weight: 20
+description: >
+  Explore Cobalt notifications.
+---
+
+{{% pageinfo %}}
+You can manage notifications for specific [pentests](#manage-pentest-notifications) and [findings](#manage-finding-notifications).
+{{% /pageinfo %}}
+
+By default, you get notifications for some activities related to your organization and pentests—both through email and in the Cobalt app. If you're not receiving notifications, make sure that you've confirmed your email address.
+
+{{%expand "View examples of activities." %}}
+<br>
+You get notified when someone has:
+
+- Created or deleted a pentest
+- Posted an update for a pentest
+- Reported a vulnerability
+- Changed a finding state
+- Invited users to the organization
+
+{{% /expand %}}
+<br>
+
+Your [user role](/platform-deep-dive/collaboration/organization/user-roles/) determines which notifications you get.
+
+## Manage Pentest Notifications
+
+When you collaborate on a pentest, you get notifications for all associated events.
```

Comment:
> When you collaborate on a pentest, you get notifications for all associated events.

Do users get notifications for **all** events? Should we say something like "for some associated events"?

Reply:
When you collaborate on a pentest, you get notifications dependent on your notification preferences.

Watching is generally ALL events
Participating is usually just mentions (e.g. @ana-dashuk-cobalt have a look at this finding) 
Ignoring = no events unless you explicitly follow a finding.